### DISCUSSION TITLE
Best way to "change to end of line"

### DISCUSSION THREAD

**Discussion by ParetoOptimalDev** [meow-edit/meow#353] [Q&A] [answered]
Cursor is `|`.

Here's a sentence with errors where it's just better to retype the rest of it:

```
A sentence is |fullll off wwords
```

With vim I would just do `C` to change until end of line and type the sentence.

With meow the best I can think of is `w;]lc`.

```
 w	  ;; meow-mark-word
 ; ;; meow-reverse
 ]	  ;; meow-end-of-thing
 l c	  ;; meow-change
```

I'm pretty sure this is the easiest way in meow, but maybe I'm missing something? I get the sense that `meow-reverse` is a constant pattern and I shouldn't be afraid I'm on the wrong path when constantly using it, right?

**Comment by DogLooksGood** [comment, marked as answer]
You mean `c$` in Vim? 

1. `meow-kill` without selection, fallback to `C-k` by default. Then switch to insert mode.
2. `meow-end-of-thing`, `l`, `meow-change`.

You don't need `meow-reverse` here.
